src/ngraph/op/experimental/dyn_broadcast.cpp

```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 //*****************************************************************************
 
+#include <numeric>
```

Comment:
(DRAFT NOTES)

- Optional input should be optional in the graph.
- Might as well move to v1 while we're here (but we can do in separate PR)